camelConnector/src/main/java/io/vantiq/extsrc/camelconn/discover/CamelResolver.java

```diff
@@ -81,6 +83,10 @@ public class CamelResolver {
             throw new IllegalArgumentException("The destination parameter cannot be null.");
         }
     
```

Comment:
Turn off a bunch of annoying logging.  Let it flow thru when we are at Trace level.